webapp/src/test/resources/derby/derby.properties

```diff
@@ -1 +0,0 @@
-derby.stream.error.logSeverityLevel=0
```

Comment:
Yes, I think git doesn't do a good job of deleting folders, but it can be deleted.  I'll give a second look.

Reply:
@karenhanson I can confirm this directory is gone, but git may not remove it from your local working copy.  You'd need to do that yourself, unfortunately (git doesn't really track directories as first-class objects, they just exist in git because they contain something)